@xen-orchestra/backups/_deltaVm.js

```diff
@@ -165,6 +175,10 @@ exports.importDeltaVm = defer(async function importDeltaVm(
     }
   }
 
+  for (const [vdiUuid, srUuid] of Object.entries(mapVdisSrs)) {
```

Comment:
this will modify the mapVdisSrs passed by address, is it the intended use ? maybe we can use a different variable since it's not the same data